golem-cli/src/worker.rs

```diff
@@ -79,12 +90,29 @@ pub enum WorkerSubcommand {
         function: String,
 
         /// JSON array representing the parameters to be passed to the function
-        #[arg(short = 'j', long, value_name = "json", value_parser = ValueParser::new(JsonValueParser))]
-        parameters: serde_json::value::Value,
+        #[arg(short = 'j', long, value_name = "json", value_parser = ValueParser::new(JsonValueParser), conflicts_with = "wave")]
+        parameters: Option<serde_json::value::Value>,
+
+        /// Function parameter in wasi-wave format
```

Comment:
```suggestion
        /// Function parameter in WAVE format
```

Just a minor thing :) the library we use is called `wasm-wave` (not related to wasi) and actually WAVE means "Web Assembly Value Encoding" so i think that's what we should refer to as a format